pkg/schedule/checker/rule_checker_test.go

```diff
@@ -168,6 +168,39 @@ func (suite *ruleCheckerTestSuite) TestFixOrphanPeers() {
 	suite.Equal(uint64(4), op.Step(0).(operator.RemovePeer).FromStore)
 }
 
+func (suite *ruleCheckerTestSuite) TestFixToManyOrphanPeers() {
+	suite.cluster.AddLeaderStore(1, 1)
+	suite.cluster.AddLeaderStore(2, 1)
+	suite.cluster.AddLeaderStore(3, 1)
+	suite.cluster.AddLeaderStore(4, 1)
+	suite.cluster.AddLeaderStore(5, 1)
+	suite.cluster.AddLeaderStore(6, 1)
+	suite.cluster.AddRegionWithLearner(1, 1, []uint64{2, 3}, []uint64{4, 5, 6})
```

Comment:
This comment should have nothing to do with this PR. But I want to know what happen if here is `suite.cluster.AddRegionWithLearner(1, 1, []uint64{2, 3}, []uint64{4})`, and other behaviors are same.

Reply:
It seems not return operator

Reply:
yes, because need len(`OrphanPeers`) >=2